src/views/taskList.html

```diff
@@ -0,0 +1,179 @@
+<main id="content" role="main">
+
+    {{> common/betaBanner}}
+
+    <div class="grid-row">
+        <div class="column-two-thirds column-minimum">
+
+            <form method="post" action="/v7/check/check-answers" name="tasklist">
+
+                <h1 class="heading-large">
+    <span class="font-medium" style="color:#6f777b">Apply for a standard rules waste permit:</span><br>
+    Anaerobic digestion facility including use of the resultant biogas (waste recovery operation)<br>
+    <span class="heading-medium">SR2012 No 12 &nbsp;
+      <a class="font-small" href="/v7/selectpermit/permit-category2">select a different permit</a>
+    </span>
+    </h1>
+
+                <ol class="task-list">
+
+                    <!-- Before you apply ############################################################### -->
+                    <li>
+                        <h2 class="task-list-section">
+              <span class="task-list-section-number">1. </span> Before you apply
+            </h2>
+                        <ul class="task-list-items">
+                            <li class="task-list-item " id="check-permit-cost-and-time">
+
+                                <a class="task-name" href="/v7/selectpermit/cost-and-time" aria-describedby="cost-and-time-completed">
+                  Check permit cost and time
+                </a>
+
+                                <input type="hidden" name="complete[]" value="Check permit cost and time">
```

Comment:
All fields in a view need an ID. Its not essential for application, but it makes writing the automated acceptance tests so much easier.

If this means we need to manually set this when creating the view, we can hold off for now until we've given it some thought. Else if there is a quick way of adding this feature, that results in static ID's (its no use if they will change on every GET request) it'd be worth doing it now.